pkg/cluster/util.go

```diff
@@ -594,3 +594,12 @@ func trimCronjobName(name string) string {
 	}
 	return name
 }
+
+func parseResourceRequirements(resourcesRequirement v1.ResourceRequirements) (acidv1.Resources, error) {
+	var resources acidv1.Resources
+	resourcesJSON, _ := json.Marshal(resourcesRequirement)
```

Comment:
so the marshalling error is ignored but the unmarshalling on the line below is not ?